drivers/sensor/Kconfig

```diff
@@ -85,6 +85,101 @@ source "drivers/sensor/dht/Kconfig"
+source "drivers/sensor/lis2dh/Kconfig"
+source "drivers/sensor/lis2ds12/Kconfig"
+source "drivers/sensor/lis2dw12/Kconfig"
+source "drivers/sensor/lis2mdl/Kconfig"
+source "drivers/sensor/lis3mdl/Kconfig"
+source "drivers/sensor/lm75/Kconfig"
+source "drivers/sensor/lm77/Kconfig"
+source "drivers/sensor/lps22hb/Kconfig"
+source "drivers/sensor/lps22hh/Kconfig"
+source "drivers/sensor/lps25hb/Kconfig"
+source "drivers/sensor/lsm303dlhc_magn/Kconfig"
+source "drivers/sensor/lsm6ds0/Kconfig"
+source "drivers/sensor/lsm6dsl/Kconfig"
+source "drivers/sensor/lsm6dso/Kconfig"
+source "drivers/sensor/lsm9ds0_gyro/Kconfig"
+source "drivers/sensor/lsm9ds0_mfd/Kconfig"
+source "drivers/sensor/max17055/Kconfig"
+source "drivers/sensor/max17262/Kconfig"
+source "drivers/sensor/max30101/Kconfig"
+source "drivers/sensor/max31855/Kconfig"
+source "drivers/sensor/max31875/Kconfig"
+source "drivers/sensor/max44009/Kconfig"
+source "drivers/sensor/max6675/Kconfig"
+source "drivers/sensor/mchp_tach_xec/Kconfig"
+source "drivers/sensor/mcp9808/Kconfig"
+source "drivers/sensor/mhz19b/Kconfig"
+source "drivers/sensor/mpr/Kconfig"
+source "drivers/sensor/mpu6050/Kconfig"
+source "drivers/sensor/mpu9250/Kconfig"
+source "drivers/sensor/ms5837/Kconfig"
+source "drivers/sensor/ms5607/Kconfig"
+source "drivers/sensor/nrf5/Kconfig"
+source "drivers/sensor/opt3001/Kconfig"
+source "drivers/sensor/pms7003/Kconfig"
+source "drivers/sensor/qdec_mcux/Kconfig"
+source "drivers/sensor/qdec_nrfx/Kconfig"
+source "drivers/sensor/qdec_sam/Kconfig"
+source "drivers/sensor/qdec_stm32/Kconfig"
+source "drivers/sensor/sbs_gauge/Kconfig"
+source "drivers/sensor/sgp40/Kconfig"
+source "drivers/sensor/shtcx/Kconfig"
+source "drivers/sensor/sht3xd/Kconfig"
+source "drivers/sensor/sht4x/Kconfig"
+source "drivers/sensor/si7006/Kconfig"
+source "drivers/sensor/si7055/Kconfig"
+source "drivers/sensor/si7060/Kconfig"
+source "drivers/sensor/si7210/Kconfig"
+source "drivers/sensor/sm351lt/Kconfig"
+source "drivers/sensor/sngcja5/Kconfig"
+source "drivers/sensor/stm32_temp/Kconfig"
+source "drivers/sensor/stm32_vbat/Kconfig"
+source "drivers/sensor/stts751/Kconfig"
+source "drivers/sensor/sx9500/Kconfig"
+source "drivers/sensor/th02/Kconfig"
+source "drivers/sensor/tmp007/Kconfig"
+source "drivers/sensor/tmp108/Kconfig"
+source "drivers/sensor/tmp112/Kconfig"
+source "drivers/sensor/tmp116/Kconfig"
+source "drivers/sensor/vcnl4040/Kconfig"
+source "drivers/sensor/vl53l0x/Kconfig"
+source "drivers/sensor/vl53l1x/Kconfig"
+source "drivers/sensor/nxp_kinetis_temp/Kconfig"
+source "drivers/sensor/wsen_hids/Kconfig"
+source "drivers/sensor/wsen_itds/Kconfig"
+source "drivers/sensor/wsen_tids/Kconfig"
+source "drivers/sensor/mcux_acmp/Kconfig"
+source "drivers/sensor/nuvoton_tach_npcx/Kconfig"
+source "drivers/sensor/nuvoton_adc_cmp_npcx/Kconfig"
+source "drivers/sensor/ite_tach_it8xxx2/Kconfig"
+source "drivers/sensor/ite_vcmp_it8xxx2/Kconfig"
+source "drivers/sensor/pcnt_esp32/Kconfig"
```

Comment:
Looks like there were some rebasing problems

Reply:
Oh Yeah. Not sure how that happened. Fixed in pull-request update. Thanks!